ranger-trino-plugin-shim/src/main/java/org/apache/ranger/authorization/trino/authorizer/RangerSystemAccessControl.java

```diff
@@ -517,18 +518,6 @@ public void checkCanShowRoleGrants(SystemSecurityContext context) {
     }
   }
 
-  @Override
-  public Optional<ViewExpression> getRowFilter(SystemSecurityContext context, CatalogSchemaTableName tableName) {
```

Comment:
Hi, @jiaoqingbo I have a question. Why did you remove this function to make ranger plugin work w/ Trino 410+?

Thanks.